go/private/context.bzl

```
@@ -187,14 +186,15 @@ def _builder_args(go, command = None, use_path_mapping = False):
         # Use a file rather than goroot as the latter is just a string and thus
         # not subject to path mapping.
         args.add_all("-goroot", [goroot_file], map_each = _dirname, expand_directories = False)
-    args.add("-installsuffix", installsuffix(go.mode))
-    args.add_joined("-tags", go.mode.tags, join_with = ",")
+    mode = go.mode
+    args.add("-installsuffix", installsuffix(mode))
+    if mode.tags:
+        args.add_joined("-tags", mode.tags, join_with = ",")
```

```suggestion
    args.add_joined("-tags", mode.tags, join_with = ",")
```

Bazel already has an early return for this case in `args.add_joined`.

---

I saw it show up but it was only 10ms and inconsistent. I'll revert this bit

---

I hit commit :⁠-⁠)

---

Ok thanks!